docs/Features/DTDFeatures.md

```diff
@@ -0,0 +1,29 @@
+# DTD Features
+
+## CodeLens Support
+
+CodeLens is supported for `.dtd` files, which can show where a type is referenced.
+
+![CodeLens DTD](../images/Features/CodeLensDTD.gif)
+
+This feature also supports listing and highlighting the number of references to a type and the line(s) referenced on.
+
+![CodeLens References DTD ](../images/Features/CodeLensReferencesDTD.gif)
+
+### Read more
+
+See [CodeLens](https://github.com/redhat-developer/vscode-xml/blob/master/docs/CodeLens.md) for more details.
```

Comment:
If you use the link `[CodeLens](CodeLens.md#code-lens)`, then the link will work on GitHub and in the built-in documentation. You need to point to a heading in the markdown file when you do this (this is a limitation of the code that runs the built-in documentation).